sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient_helpers.cs

```diff
@@ -6,6 +6,7 @@
 using System.Linq;
 using System.Text;
 using System.Text.Json;
+using System.Text.RegularExpressions;
```

Comment:
I didn't find any regex functionality be used in this file

Reply:
nice catch